scripts/portal.js

```diff
@@ -137,14 +137,8 @@ function Portal(url)
     if(this.json.client_version){
       // Used to check if the rotonde version matches when multiple version lines are present.
       var client_version_main = this.json.client_version.split(/\r\n|\n/)[0];
-      // Based off of entry.formatter
-      this.json.client_version = this.json.client_version
-        .replace(/&/g, "&amp;")
-        .replace(/</g, "&lt;")
-        .replace(/>/g, "&gt;")
-        .replace(/"/g, "&quot;")
-        .replace(/'/g, "&#039;")
-        .split(/\r\n|\n/).slice(0, 2).join("<br>") // Allow 2 lines for mod versions
+      this.json.client_version = r.escape_html(this.json.client_version)
```

Comment:
I just realized that this causes issues when using `\n` for mod versions because the `<br>` gets re-escaped to `&lt;br&gt;`. A fix would be to store the escaped result in a temporary variable instead.

Reply:
hmm, do we *need* `\n` in version numbers? how about a simple `/`

Reply:
actually, this (or some other glyph, maybe `/`) could possibly be parsed as newline in the interface, if that happens to look better

Reply:
`\n` isn't necessary at all, but I just added support for that before the version line gets too wide.  
Good point about other separators - an alternative to `var client_version_main = this.json.client_version.split(/\r\n|\n/)[0]` would be to use the match of `/^[0-9.]+/` (or even `^[0-9.]+[a-z]?` as preparation for releases with alphabetic suffixes)